packs/magento.json

```diff
@@ -5,7 +5,7 @@
   "total_byte_weight": "Disable Magento's built-in [JavaScript bundling and minification](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/js-bundling.html), and consider using [baler](https://github.com/magento/baler/).",
   "render_blocking_resources": "Disable Magento's built-in [JavaScript bundling and minification](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/js-bundling.html), and consider using [baler](https://github.com/magento/baler/) instead.",
   "unminified_css": "Enable the \"Minify CSS Files\" option your store's Developer settings. [Learn more](https://devdocs.magento.com/guides/v2.3/performance-best-practices/configuration.html?itm_source=devdocs&itm_medium=search_page&itm_campaign=federated_search&itm_term=minify%20css%20files).",
-  "unminified_javascript": "Use [Terser](https://www.npmjs.com/package/terser) to minify all JavaScript assets outfrom from static content deployment, and disable the built-in minification feature.",
+  "unminified_javascript": "Use [Terser](https://www.npmjs.com/package/terser) to minify all JavaScript assets from static content deployment, and disable the built-in minification feature.",
```

Comment:
good catch @samdutton!

question for the original author (??)  but .. what are "all javascript assets from static content deployment"?  and how does that differentiate from "all JS assets"

Reply:
CC @DrewML. Should we be specific here when we say "from static content deployment"?